pricing-page-exercise/salih/styles.css

```diff
@@ -0,0 +1,195 @@
+/* Global */
+
+*, ::after, ::before {
+    box-sizing: border-box;
+    font-family:sans-serif;
+}
+ul{
+    list-style-type: none;
+    padding: 0;
+    text-align: center;
+}
+.main li{
+    padding: 0.2rem 0;
+}
+h1{
+    margin: 0;
+    text-align: center;
+    margin-top: 5px;
+}
+button{
+    margin: 15px 10px;
+    min-width: 60%;
+    padding: 12px 0;
+}
+small{
+    font-weight: 100;
+    font-size: 0.5em;
+}
+.blue-button{
+    background-color: #007bff;
```

Comment:
Try using only one type of colors (HEX is more commonly used).

Reply:
I changed to all HEX code except the one which need to be transparent.